### DISCUSSION TITLE
Recent update bird identification all wrong?

### DISCUSSION THREAD

**Discussion by malone-patpat** [mcguirepr89/BirdNET-Pi#1028] [Q&A]
I just recently did an update to my birdnet-pi and now all the bird identifications are just wrong.  It's saying I have tropical parrots and eagles and all kinds of birds that are not here but shows none of my usual birds.  Listening to some recordings, what is clearly a chickadee is identified as a white bellied parrot and a flicker as a pied cuckoo.  Any idea how I can fix this?  I suppose I could just re-install from scratch.  Or could something be off with my microphone? (It was not picking up anything so I did the update and then re-did all the connections - it has more buzz/hiss than I remember)

**Comment by ehpersonal38** [comment]
Go to Tools -> Settings -> Model, and switch the model to the one you're not currently using, Update Settings at the bottom of the page, then repeat the steps except this time selecting the model you actually want. This should reset the model/labels.txt.

**Comment by gosssamer** [comment]
This appears to have made a significant improvement. Thanks so much.

**Comment by huminator1978** [comment]
Can you adjust the recorded bird species on Raspberry Pi?

**Reply by alexbelgium** [reply]
What do you mean? If you mean change a bird detection to another bird, you should use this version of Birdnet-Pi which supports the feature (and is more up-to-date) : https://github.com/Nachtzuster/BirdNET-Pi

If you mean tweak which species can be detected by your system, you should adapt the type of model, and threshold of presence probability. For example, bird of preys have a very low presence probability due to how those values were defined. Once again, the fork above provides error messages in the log that explain why a specific specie was discarded even if its confidence is greater than the setting : it is usually due to this threshold, or it can be because it is in the "excluded species list".
